### PR TITLE
Adding Spanish files

### PR DESCRIPTION
**Pull request by rafaecheve**

### CODE REVIEW

File: es/ios/analytics.mdown [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Analytics
+
+Parse provee un numero de herramientas para verificar la actividad de tu aplicación. Comprendemos que es importante el entender como le esta llendo a tu aplicación, que tan frecuente es la actividad, y cuando esta sucediendo.
+
+Mientras esta sección cubrira diversas maneras de utilizar tu aplicación de la mejor forma para aprovechar los analytics de Parse en el Backend, los desarrolladores usando Parse para guardar y recibir información pueden tomar ventaja de usar metricas en Parse al momento.
+
+Sin tener la nesecidad de implementar codigo del lado de cliente, puedes ver en tiempo real graficas y detalles (por  dispositivo, clase de Parse, o verbo REST) de tu peticiones API en la consola de aplicaciones, ademas guardar filtros de graficas, para accesar a la información de tu interés.
```
Comment (hramos) [inline es/ios/analytics.mdown]: Necesidad, gráficas, "tus peticiones"

File: es/ios/analytics.mdown [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+```
+```swift
+// en iOS
+PFAnalytics.trackAppOpenedWithLaunchOptions(launchOptions)
+
+// en OS X
+PFAnalytics.trackAppOpenedWithLaunchOptions(nil)
+```
+
+Graficas y detalles de tus estadisticas son accesibles desde la consola de tu aplicación.
```
Comment (hramos) [inline es/ios/analytics.mdown]: Gráficas

File: es/ios/analytics.mdown [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Mas metricas estan disponibles desde la entrega de notificaciones push y accesos. Asegurate de mirar [Monitoreo de Push y Lanzamientos de aplicación](/docs/push_guide#receiving-tracking/iOS)
```
Comment (hramos) [inline es/ios/analytics.mdown]: Más

File: es/ios/analytics.mdown [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+## Custom Analytics
+
+`PFAnalytics` tambien permire hacer monitoreo de forma libre, con la ayuda de `NSString` atravez de llaves y valores. Estas dimenciones extra, se permite hacer segmentacion de tus eventos propios atraves de la consola de aplicaciones
```
Comment (hramos) [inline es/ios/analytics.mdown]: permite

File: es/ios/config.mdown [hunk 5]
```diff
@@ -0,0 +1,111 @@
+# Config
+
+## Parse Config
+
+`PFConfig` is una manera de configurar tus aplicaciones de manera remota permitiendote almacenar un archivo de configuracion en Parse. Permite agregar cosas como acceso a feateures para pruebas o simplemente un mensaje del dia. Para comenzar a usar `PFConfig` nesecitas agregar algunos pares de valores/llave (parametros) a la consola de Parse Config.
```
Comment (hramos) [inline es/ios/config.mdown]: configuración
s/feateures/características

File: es/ios/config.mdown [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+  if let welcomeMessage = welcomeMessage {
+    println("Welcome Message = \(welcomeMessage)!")
+  } else {
+    println("Falling back to default message.")
+    welcomeMessage = "Welcome!";
+  }
+};
+```
+
+## Actual Config
```
Comment (hramos) [inline es/ios/config.mdown]: Configuración Existente

File: es/ios/config.mdown [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+*   NSString
+*   NSNumber
+*   NSDate
+*   PFFile
+*   PFGeoPoint
+*   NSArray
+*   NSDictionary
+
+Actualmente se permiten hasta **100** parametro en tu config con un total de **128KB** contanto todos los parametros.
```
Comment (hramos) [inline es/ios/config.mdown]: contando

File: es/ios/crash-reporting.mdown [hunk 8]
```diff
@@ -0,0 +1,145 @@
+# Reporte de Errores
+
+El Reporte de Errores te permite saber como tu aplicacion esta haciendo presentando errores alla afuera. Seras capaz de ver los que causan mas impacto, con sus stack traces, información del dispositivo y mas. Para comenzar, dirijase a [Quick Start](/apps/quickstart#analytics/crashreporting/ios/existing) para instrumentar tu aplicación.
+
+
+## Habilitandolo
+
+Para habilitar el reporte de errors, agrega esta linea a tu AppDelegate antes de inicilaizar tus llaves de Parse.
```
Comment (hramos) [inline es/ios/crash-reporting.mdown]: inicializar

File: es/ios/crash-reporting.mdown [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+// Setup Parse
+Parse.setApplicationId("parseAppId", clientKey: "parseClientKey")
+```
+Una vez actualizado, todos los errores de tu app seran enviandos a Parse y se mostraran en el panel de analytics.
+
+## Simbolización
+
+Es importante enviar a Parse los archivos symbol para cada build de tu app. Esto permitira a Parse propiamente delegar y agregar los incidentes de errores para unirlos y mostrarlos en el dashboard con simbolos propios en el stack trace.  
+
+Existen dos maneras para subir tus  archivos symbol: automaticamente despues de cada build en Xcode (view instructions in [Quick Start](/apps/quickstart#analytics/crashreporting/ios/existing)) y manualmente. Recomendamos hacerlo automaticamente, para asi no tengas que recordar hacerlo cada vez al terminar un release.
```
Comment (hramos) [inline es/ios/crash-reporting.mdown]: ver instrucciones en el QuickStart

File: es/ios/crash-reporting.mdown [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+*   Add a new **Run Script** phase to the **Build Phases** of your App's target:
+
+```bash
+export PATH=/usr/local/bin:$PATH
+cd <path_to_cloudcode_folder>
+
+parse symbols "${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}"
+```
+
+Alternaticamente, puedes subir los archivos symbol manualmente utilizando la herramienta de linea de comandos:
```
Comment (hramos) [inline es/ios/crash-reporting.mdown]: Alternativamente

File: es/ios/crash-reporting.mdown [hunk 11]
```diff
@@ -0,0 +1,145 @@
...
+
+3.  _Incremente la version de tu build (CFBundleVersion) y compila un nuevo release._
+
+4.  _Marca el error como resuelto y lanza un nuevo build al App Store._ Una vez resuelto, el error no aparecera en la pagina del listado de errores con los filtros por default. si el error ocurre de nuevo en una version posterior, sera automaticamente marcado como no resuelto, y se mostrara en el listado de nuevo. 
+
+5.  Regresa al punto numero 1 y continua mejorando tu app!
+
+## Troubleshooting
+
+### ¿Porque mis crashes no se muestran en el dashboard?
```
Comment (hramos) [inline es/ios/crash-reporting.mdown]: Por qué

File: es/ios/crash-reporting.mdown [hunk 12]
```diff
@@ -0,0 +1,145 @@
...
+
+Tu app probablemente no esta encuando los reportes de errores a Parse. Aqui algunos cosas a verificar:
```
Comment (hramos) [inline es/ios/crash-reporting.mdown]: Aquí

File: es/ios/crash-reporting.mdown [hunk 13]
```diff
@@ -0,0 +1,145 @@
...
+
+*   Make sure you've enabled Crash Reporting in your App Delegate before you initialize Parse.
```
Comment (hramos) [inline es/ios/crash-reporting.mdown]: These lines are already translated below, I'm guessing you left these here by mistake.

File: es/ios/crash-reporting.mdown [hunk 14]
```diff
@@ -0,0 +1,145 @@
...
+*   Make sure you don't have any other crash reporting solutions linked to your application, as they might interfere with each other.
+
+*   Asegurate de haber habilitado el Reporte de Errores en el App Delegate antes de inicializar Parse.
+*   Si estas haciendo pruebas, asegurate que Xcode no esta haciendo Cache del error con el debugger.
+*   Los errores son enviando  cuando se corra el app por en consiguiente. Asegurate de que el app no es tiene errores que impidan el enviar el informe del reporte de errores. Si estas aun probando, puedes asegurarte agregando un retraso.
+*   Los errores pueden tomar hasta un minuto para mostrarse en el dashboard.
+*   Make sure you don't have any other crash reporting solutions linked to your application, as they might interfere with each other.
+*   Asegurate de no tener ninguna otra solución para reporte de errores vinculada a tu aplicación, que probablemente puedan intervenir una con otra.
+
+### ¿Porque mis errores no estan simbolizados?
```
Comment (hramos) [inline es/ios/crash-reporting.mdown]: Por qué

File: es/ios/crash-reporting.mdown [hunk 15]
```diff
@@ -0,0 +1,145 @@
...
+
+Probablemente no has subido los archivos symbol para el build de al app asociada con el error. Aqui algunas cosas a verificar:
+
+*   If you set up automatic uploading, verify that the uploads have worked by looking at the logs under the Report navigator.
```
Comment (hramos) [inline es/ios/crash-reporting.mdown]: Ditto, these lines can be removed.

File: es/ios/getting-started.mdown [hunk 16]
```diff
@@ -0,0 +1,7 @@
+# Comenzando
+ 
+Si no ha instalado el SDK todavía, [por favor diríjase a la guía de inicio rápido](/apps/quickstart)  para obtener el SDK para Xcode. Tenga en cuenta que soportamos iOS 6.0 y superior. También puede consultar nuestra [API de referencia para iOS](/docs/ios/api) y [para OSX](/docs/osx/api) para obtener información más detallada acerca de nuestro SDK.
+ 
+La plataforma de Parse proporciona una solución de back-end completa para su aplicación móvil . Nuestro objetivo es eliminar totalmente la necesidad de escribir código de servidor o el mantenimiento de servidores.
+ 
+En Parse , se crea una aplicación para cada una de sus aplicaciones móviles. Cada aplicación tiene su propio ID de aplicación y la clave cliente (client key ) que se utiliza en su SDK al instalar. Su cuenta en Parse puede acomodar múltiples aplicaciones. Esto es útil cuando tiene una aplicación, ya que se puede implementar diferentes versiones de prueba y producción.
```
Comment (hramos) [inline es/ios/getting-started.mdown]: Remove space between Parse and ,

**Comment by hramos** [comment]
Some nits.

**Comment by hramos** [comment]
This is great! Can you squash all of this into a single commit?

**Comment by rafaecheve** [comment]
Great I will do that, once I get more files translated I will add them in a single commit

**Pull request merged**